Text/CustomEntitySearch/CustomEntitySearch.cs

```diff
@@ -93,5 +95,146 @@ public static class CustomEntitySearch
             return new OkObjectResult(response);
         }
 
+        private static void AddValues(string checkMatch, string text, List<Entity> entities, HashSet<string> entitiesFound, int leniency)
+        {
+            bool addWord = false;
+            string escapedWord = Regex.Escape(checkMatch);
+            string pattern = @"\b(?ix:" + escapedWord + @")\b";
+            MatchCollection entityMatch = Regex.Matches(text, pattern, RegexOptions.IgnoreCase, TimeSpan.FromSeconds(MaxRegexEvalTime));
+            if (entityMatch.Count != 0)
+            {
+                foreach (Match match in entityMatch)
+                {
+                    entities.Add(
+                        new Entity
+                        {
+                            Category = "customEntity",
+                            Value = match.Value,
```

Comment:
whats the purpose of this field if its always hardcoded to customEntity?

Reply:
The purpose of this field is to provide the "same" json output as NER skill. As there is no categorization in my code, I hardcoded it to customEntity

Reply:
do we need the same json schema across these two skills? I understand using it as a starting point to keep it similar, but if category doesn't provide value in this skill there's no reason to include it

---
In reply to: [312108230](https://github.com/Azure-Samples/azure-search-power-skills/pull/5#discussion_r312108230) [](ancestors = 312108230)